src/models/config.py

```diff
@@ -0,0 +1,100 @@
+"""Dataclasses for the prefect deployments"""
```

Comment:
Is this only related to prefect? Then it would probably belong into that repo.

Reply:
Sorry, I forgot to change the comment at the top. I moved this to the dune-sync repo since I thought it would be more suitable to hold chain/env specific logic here. But looking at the code again I think you're right. If we refactor this to be chain and env agnostic then these configs won't have much logic anyway.

Do you think we'll be setting specific logic in the dune-sync repo based on env/chain?

Reply:
If I understood correctly, starting with dune-sync v2, we will not have any cow protocol related code in that repo. All config related to our infrastructure (e.g. prod/barn, which queries to use, how infra changes are tested) will not be part of dune-sync.

You could also have a look at https://github.com/bh2smith/dune-sync to see what the current plan. Providing feedback over there if something is missing might also be useful.

Reply:
Thanks, I'll close this PR and also look at that repo.